subprojects/docs/src/docs/release/notes.md

```diff
@@ -184,6 +184,10 @@ See the User Manual section on the “[Feature Lifecycle](userguide/feature_life
 
 The following are the features that have been promoted in this Gradle release.
 
+### Shared Build Services
+
+[Shared Build Services](current/userguide/build_services.html) is promoted to a stable feature.
```

Comment:
:x: this link won't work, it should be `[Shared Build Services](userguide/build_services.html)` instead

Reply:
Thanks, @eskatos. Fixed in https://github.com/gradle/gradle/pull/19483.